app/renderer/components/tabs/content/privateIcon.js

```diff
@@ -20,18 +23,23 @@ class PrivateIcon extends React.Component {
   mergeProps (state, ownProps) {
     const currentWindow = state.get('currentWindow')
     const frameKey = ownProps.frameKey
+    const hasSeconardImage = tabContentState.hasVisibleSecondaryIcon(currentWindow, ownProps.frameKey)
```

Comment:
misspelling here; s/hasSeconardImage/hasSecondaryImage/g

Reply:
yep that was just a move of what we had at that moment and method was eventually overridden